src/explorer/pagerankTable/TableRow.test.js

```diff
@@ -34,9 +55,29 @@ describe("explorer/pagerankTable/TableRow", () => {
           children={<div data-test-children={true} />}
         />
       );
-      const trStyle = el.find("tr").props().style;
+
       const buttonStyle = el.find("button").props().style;
-      expect({depth, trStyle, buttonStyle}).toMatchSnapshot();
+      const trClassName = el.find("tr").props().className;
+      expect({depth, trClassName, buttonStyle}).toMatchSnapshot();
+
+      if (depth === 0) {
+        const expectedClassName = tableRowStyle.highlightOnHover._name;
+        expect(trClassName).toBe(expectedClassName);
+      } else if (depth === 1) {
+        const backgroundColor = "hsla(150,100%,28%,0.09999999999999998)";
+        const expectedClassName = setBackgroundColor(backgroundColor);
+        expect(trClassName).toBe(expectedClassName._name);
+        expect(expectedClassName._definition.backgroundColor).toBe(
+          backgroundColor
+        );
+      } else if (depth === 2) {
+        const backgroundColor = "hsla(150,100%,28%,0.18999999999999995)";
+        const expectedClassName = setBackgroundColor(backgroundColor);
+        expect(trClassName).toBe(expectedClassName._name);
+        expect(expectedClassName._definition.backgroundColor).toBe(
+          backgroundColor
+        );
+      }
```

Comment:
If none of these cases is hit, the test will silently pass. Someone
could change the loop above from `[0, 1, 2]` to `[0, 1, 2, 3]`, but the
additional invocation wouldn’t test anything with respect to styling.

Two solutions that come to mind:

  - add an `else` clause with `throw new Error("Should be unreachable")`
    or similar;

  - refactor the loop to be over an array of objects like

    ```js
    [
      {depth: 0, expectedAlpha: …},
      …,
    ]
    ```